modulemd/modulemd-docs.xml

```diff
@@ -79,6 +80,7 @@
        <xi:include href="xml/modulemd-subdocument-info-private.xml"/>
        <xi:include href="xml/modulemd-translation-private.xml"/>
        <xi:include href="xml/modulemd-translation-entry-private.xml"/>
+       <xi:include href="xml/modulemd-eol-private.xml"/>
```

Comment:
Please also add `xml/api-index-2.10.xml` below, or these functions will not appear in the index.